### PR TITLE
Display an error when updating .htaccess failed

### LINKED ISSUE #1129: maintenance:update:htaccess doesn't throw error when htaccess is read-only
### Steps to reproduce
1. change ownership of htaccess to root
2. call `sudo -u www-data php occ maintenance:update:htaccess`

### Expected behavior
Nextcloud should throw an error

### Actual behavior
Nextcloud says:
`.htaccess has been updated`

### Server configuration
**Operating system**: ubuntu 16.04

**Web server:** apache2

**Database:** sqlite

**PHP version:** 7

**Nextcloud version:** 10

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #1129 

@georgehrke @LukasReschke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Let me rebase this.

File: lib/private/Setup.php [hunk 1]
```diff
@@ -472,9 +472,10 @@ public static function updateHtaccess() {
 
 		if ($content !== '') {
 			//suppress errors in case we don't have permissions for it
-			@file_put_contents($setupHelper->pathToHtaccess(), $htaccessContent.$content . "\n");
+			return (bool) @file_put_contents($setupHelper->pathToHtaccess(), $htaccessContent.$content . "\n");
```
Comment (LukasReschke) [inline lib/private/Setup.php]: Update PHPDoc? :)

Comment (nickvergessen) [inline lib/private/Setup.php]: added

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by MorrisJobke** [comment]
@rullzer easy to review :)

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**